### DISCUSSION TITLE
quick typo fix:

### DISCUSSION THREAD

**Discussion by liamzee** [haskell-github-trust/.github#8] [General]
```
The essential features of the Haskell GitHub Trust are

All Haskell Github Trust organization members are Owners, and have control over all repositories, including tranferring in and out.
A Hackage “Group Account” [haskell_github_trust](https://hackage.haskell.org/user/haskell_github_trust).
```

"tranferring in and out"

**Comment by liamzee** [comment]
Thanks for being here! I hope your trust organization grows quickly so that it's viable to get certain unmaintained packages put into trust (or at least trust assistance) because we lose Haskellers due to the difficulty of keeping up with GHC!

**Comment by jamesdbrock** [comment]
Thanks! Fixed.
